cubic-server/protocol/ServerPackets.hpp

```diff
@@ -26,6 +26,7 @@ namespace protocol
         QueryBlockEntityTag = 0x01,
         ChangeDifficulty = 0x02,
         ChatMessage = 0x03,
+        ChatCommand = 0x04,
```

Comment:
0x03, we are still in 1.19 (I think)
And yes, ChatMessage should be 0x04, but that does not fix the chat...